pkgs/applications/office/imag/default.nix

```diff
@@ -0,0 +1,37 @@
+{ stdenv
+, lib
+, rustPlatform
+, fetchFromGitHub
+, llvmPackages
+, openssl
+, pkg-config
+, Security
+}:
+
+rustPlatform.buildRustPackage rec {
+  pname = "imag";
+  version = "0.10.0";
+
+  src = fetchFromGitHub {
+    owner = "matthiasbeyer";
+    repo = pname;
+    rev = "v${version}";
+    sha256 = "1rkl4n3x2dk9x7705zsqjlxh92w7k6jkc27zqf18pwfl3fzz8f8p";
+  };
+
+  buildInputs = [ openssl pkg-config ]
```

Comment:
`pkg-config` is a build runtime dependency:

```diff
-  buildInputs = [ openssl pkg-config ]
+  nativeBuildInputs = [ pkg-config ];
+  buildInputs = [ openssl ]
```

https://nixos.org/nixpkgs/manual/#ssec-stdenv-dependencies